rollup-common.js

```diff
@@ -394,9 +395,9 @@ export function litProdConfig({
         sourcemaps(),
         terser(terserOptions),
         summary(),
-        ...(CHECKSIZE
-          ? [skipBundleOutput]
-          : [
+        ...(CHECKSIZE ? [skipBundleOutput] : []),
+        ...(copyHtmlTests && !CHECKSIZE
```

Comment:
I think parens is needed for this to be evaluated together before the ternary.
```suggestion
        ...((copyHtmlTests && !CHECKSIZE)
```

Reply:
Nevermind, it seems parens isn't needed.